pyramus/updates/66.001/180522_matriculation.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Has to be upped to version 68.001 due to recent database updates

Reply:
OK, thank you!